lib/mempool.go

```diff
@@ -61,38 +61,6 @@ var (
 	LowFeeTxLimitBytesPerTenMinutes = 150000 // Allow 150KB per minute in low-fee txns.
 )
```

Comment:
Maybe we should rename this file to legacy_mempool.go or something so it's clear that it will all be deleted soon.

Reply:
Hah I'm down, purge! File renamed